app/controllers/games_controller.rb

```diff
@@ -48,10 +55,12 @@ def board_display_piece_query(row, column)
 
   def place_piece_td(row, column)
     find_piece = board_display_piece_query(row, column)
-    board_square = "<td class='x-position-'#{column}' "
-    board_square += "piece-id-data='#{piece_id(find_piece)}' "
-    board_square += "piece-type-data='#{piece_type(find_piece)}''>"
-    unless find_piece.nil?
+    board_square = "<td data-x-position='#{column}'"
+    if find_piece.nil?
+      board_square += ">"
+    else
+      board_square += " data-piece-id='#{piece_id(find_piece)}' "
+      board_square += "data-piece-type='#{piece_type(find_piece)}'>"
```

Comment:
Swapped the naming of all these data attributes around....apparently you need to have them read 'data-(whatever name you want here)', and i originally had it backwards